iOS Client/Store/MBStore.swift

```diff
@@ -78,12 +86,26 @@ class MBStore: NSObject {
     
 
     //New function for devotions until we figure out if this will have to go over network or can be stored locally
-    func syncDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        loadDevotions { (devotions, devotionError) in
-            completion(devotions, devotionError)
+    func syncDevotions(completion: @escaping ([LoadedDevotion]?, Error?) -> Void) {
+        self.client.getJSONFile(name: "devotions") { data, error in
+            if error == nil, let data = data {
+                do {
+                    //We got some data now parse
```

Comment:
hells yeah we do

Reply:
could use a whitespace between second slash and We. this should be a lint warning if it's not